data-plane/tests/src/test/java/dev/knative/eventing/kafka/broker/tests/DataPlaneTest.java

```diff
@@ -98,6 +102,7 @@
   private static KafkaCluster kafkaCluster;
   private static ConsumerDeployerVerticle consumerDeployerVerticle;
   private static ReceiverVerticle receiverVerticle;
+  private static int numResources;
```

Comment:
Move this below `NUM_SYSTEM_VERTICLES`, rename it to `NUM_RESOURCES`, and make it `final` to be consistent.